src/cljs/main/broadfcui/main.cljs

```diff
@@ -112,6 +112,33 @@
                   "http://status.firecloud.org/"]
                  " for more information."])}))
 
+(defn status-alert [title message link]
+  [:div {:style {:borderBottom "1px solid" :borderBottomColor (:line-default style/colors)
+                 :backgroundColor (:exception-state style/colors) :padding "1rem"}}
+   [:div {:style {:color "#fff" :marginBottom "0.5rem" :fontWeight 600}}
+    [icons/ExceptionIcon {:size 18 :paddingRight "0.5rem"}] (if title title "Service Alert")]
+   [:div {:style {:color "#fff" :fontSize "90%"}}
+    [:div {} (str message " ")
+     (when link [:a {:href (str link) :target "_blank" :style {:color "#fff"}} "Read more..."])]]])
+
+(react/defc ShowStatusAlerts
+  {:render
+   (fn [{:keys [this state]}]
+     (let [alerts (:alerts @state)]
+       (when (not (empty? alerts))
+         [:div {}
+          (map (fn [alert]
+                 (status-alert (:title alert) (:message alert) (:link alert)))
+               alerts)])))
+   :component-did-mount
+   (fn [{:keys [state]}]
+     (utils/ajax {:url (config/alerts-json-url)
+                  :headers {"Cache-Control" "no-store, no-cache"}
+                  :on-done (fn [{:keys [raw-response]}]
+                             (let [[parsed _] (utils/parse-json-string raw-response true false)]
+                               (if (not (empty? parsed))
```

Comment:
Should this be a `when`?

See comment above on `when-let`, otherwise `if-let` exists too.